File: Exercises/Easy/ex1/Solucao.c
#include <stdio.h>

void encontrar_indices(int nums[], int tamanho, int target) {
    for (int i = 0; i < tamanho; i++) {
        for (int j = i + 1; j < tamanho; j++) {
            if (nums[i] + nums[j] == target) {
                printf("Indices encontrados: [%d, %d]\n", i, j);
                return;
            }
        }
    }
    printf("Nenhuma combinacao encontarada.\n");
}


int main() {
    int vetor[5];
    int target;

    printf("Informe o numero target: ");
    scanf("%d", &target);

    printf("Agora informe os valores dentro do vetor:\n");
    for (int i = 0; i < 5; i++) {
        printf("valor na posicao [%d]: ", i);
        scanf("%d", &vetor[i]);
    }

    int tamanho = sizeof(vetor) / sizeof(vetor[0]);
    encontrar_indices(vetor, tamanho, target);

    return 0;
}
 

File: Exercises/Easy/ex1/Solucao.py
def encontrar_indices(nums, target):
    for i in range(len(nums)):
        for j in range(i + 1, len(nums)):
            if nums[i] + nums[j] == target:
                print(f"Índices encontrados: [{i}, {j}]")
                return
    print("Nenhuma combinação encontrada.")

def main():
    target = int(input("Informe o número target: "))
    tam = int(input("Informe o tamanho do vetor:"))
    vetor = []
    print("Agora informe os valores dentro do vetor:")
    for i in range(tam):
        valor = int(input(f"Valor na posição [{i}]: "))
        vetor.append(valor)
    
    encontrar_indices(vetor, target)

if __name__ == "__main__":
    main()
